example/tests/tests.dart

```diff
@@ -676,3 +645,186 @@ Future<StringBuffer> testSynchronousDeinit() async {
 
   return StringBuffer();
 }
+
+/// Test voice groups.
+Future<StringBuffer> testVoiceGroups() async {
+  await initialize();
+
+  final currentSound =
+      await SoLoud.instance.loadAsset('assets/audio/explosion.mp3');
+
+  /// Start playing sounds in pause state to get their handles.
+  final h1 = await SoLoud.instance.play(currentSound, paused: true);
```

Comment:
I thought the same at the beginning, but groups accepts only the handles returned by the `play`. When the handle ends it is also removed from the group. It's not possible to automatically add handles of a `SoundHash` using the C++ SoLoud lib, but maybe something can be done outside it (ie a new feature to use sounds instead of handles in groups).

Reply:
Nono, this is fine, then. Let's keep close to the original API in this case.